packages/dashboard-frontend/src/store/DevfileRegistries/const.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2018-2024 Red Hat, Inc.
```

Comment:
2025

Reply:
It depends on the [copyright.js](https://github.com/eclipse-che/che-dashboard/blob/main/.config/copyright.js) file, which includes the possible license:

```
/*
 * Copyright (c) 2018-2024 Red Hat, Inc.
 * This program and the accompanying materials are made
 * available under the terms of the Eclipse Public License 2.0
 * which is available at https://www.eclipse.org/legal/epl-2.0/
 *
 * SPDX-License-Identifier: EPL-2.0
 *
 * Contributors:
 *   Red Hat, Inc. - initial API and implementation
 */
```

It should be a separate PR that changes the license file and a fix file license for the whole project.